src/ServiceControl.Monitoring/Infrastructure/HistoryPeriod.cs

```diff
@@ -8,36 +8,35 @@ public class HistoryPeriod
     {
         static HistoryPeriod()
         {
-            All = new List<HistoryPeriod>()
+            All = new List<HistoryPeriod>
             {
-                new HistoryPeriod(TimeSpan.FromMinutes(5)),
-                new HistoryPeriod(TimeSpan.FromMinutes(10)),
-                new HistoryPeriod(TimeSpan.FromMinutes(15)),
-                new HistoryPeriod(TimeSpan.FromMinutes(30)),
-                new HistoryPeriod(TimeSpan.FromMinutes(60))
+                new HistoryPeriod(TimeSpan.FromMinutes(1), 40),
```

Comment:
This results in 1.5 second buckets. Was 40 buckets just an arbitrary doubling of the previous number? Would it make sense to have 30 2-second buckets or even 60 1-second buckets? Do does that mess with the UI?

Reply:
This is a good point. What we want to do is get on a call and tweak UI and SC.Montoring at the same time to get best results. 

Until than I keep this as `WIP`